cpu/nrf52/radio/nrf802154/nrf802154_radio.c

```diff
@@ -265,6 +270,10 @@ static int _confirm_op(ieee802154_dev_t *dev, ieee802154_hal_op_t op, void *ctx)
 
         state = STATE_IDLE;
         enable_shorts = true;
+        if (info) {
```

Comment:
Why is this the right location for this check and not just before `if (eagain)` (below switch case?)

Reply:
because that overwrites the `IEEE802154_HAL_OP_CCA` case. Since `info` has "something", it replaces the CCA status with garbage

Reply:
Sorry, I still don't fully understand. 
1. What do you mean by "that" overwrites the `IEEE802154_HAL_OP_CCA` case? How to overwrite a case at all?
2. The starting point for my comment was: we are in the "confirm operation" function here and I was wondering, shouldn't be the "confirm CCA operation" actually be the most accurate place to indicate whether the CCA found a busy or free channel?

Reply:
The `IEEE802154_HAL_OP_CCA` already handles `ctx`, in fact it always assumes it to be a `bool *` that must be present.

If `info` is then set *after* the switch case, it would overwrite that.